### PR TITLE
SE-3151: Term Deposit Notebook

### PR DESCRIPTION
**Pull request by liberty-askew**

### CODE REVIEW

File: examples/use-cases/instruments/Term Deposit.ipynb [hunk 1]
```diff
@@ -0,0 +1,1668 @@
+{
```
Comment (liberty-askew) [inline examples/use-cases/instruments/Term Deposit.ipynb:1]: <p>Add parameter names</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (liberty-askew) [inline examples/use-cases/instruments/Term Deposit.ipynb:1]: <pre class="ql-syntax" spellcheck="false">  Line #16.                    lm.MarketDataKeyRule(

remove this
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (liberty-askew) [inline examples/use-cases/instruments/Term Deposit.ipynb:1]: <p>Rolling full term deposit maturity amount</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (liberty-askew) [inline examples/use-cases/instruments/Term Deposit.ipynb:1]: <p>Change this to 4%</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (liberty-askew) [inline examples/use-cases/instruments/Term Deposit.ipynb:1]: <p>Change this to use cashflow value not valuation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (liberty-askew) [inline examples/use-cases/instruments/Term Deposit.ipynb:1]: <p>Rolling the notional for the term deposit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Comment (liberty-askew) [inline examples/use-cases/instruments/Term Deposit.ipynb:1]: <p>Add view strnasactions cell </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

File: examples/use-cases/instruments/Term Deposit.ipynb [hunk 2]
```diff
@@ -0,0 +1,1458 @@
...
```
Comment (slemasne) [inline examples/use-cases/instruments/Term Deposit.ipynb:1]: <p>Rerun so you get HTML output without errors. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (slemasne) [inline examples/use-cases/instruments/Term Deposit.ipynb:1]: <p>Should there be output here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

File: examples/use-cases/instruments/Term Deposit.ipynb [hunk 3]
```diff
@@ -0,0 +1,1709 @@
...
```
Comment (slemasne) [inline examples/use-cases/instruments/Term Deposit.ipynb:1]: <p>Why does B have a valuation of 0? Does it need to be matured? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>

**Review by slemasne — APPROVED** [review]

**Pull request merged**